bigtable/examples/run_examples_utils.sh

```diff
@@ -19,6 +19,47 @@ readonly CBT_CMD="${CBT:-${GOPATH}/bin/cbt}"
 readonly CBT_EMULATOR_CMD="${CBT_EMULATOR:-${GOPATH}/bin/emulator}"
 readonly CBT_INSTANCE_ADMIN_EMULATOR_CMD="../tests/instance_admin_emulator"
 
+function cleanup_instance {
+  local project=$1
+  local instance=$2
+  shift 2
+
+  if [ -z "${project}" -o -z "${instance}" ]; then
```

Comment:
This should be split into independent tests for `$1` and `$2` above.

Reply:
Ack.